addons/medical_statemachine/stringtable.xml

```diff
@@ -8,33 +8,17 @@
             <Russian>Состояния</Russian>
             <French>États</French>
         </Key>
-        <Key ID="STR_ACE_Medical_Statemachine_FatalInjuryCondition_DisplayName">
-            <English>Fatal Injury Player</English>
-            <German>Tödliche Spielerverletzungen</German>
-            <Japanese>致命傷プレイヤー</Japanese>
-            <Russian>Смертельные ранения игрока</Russian>
-            <French>Blessure fatale des joueurs</French>
+        <Key ID="STR_ACE_Medical_Statemachine_FatalInjuriesPlayer_DisplayName">
+            <English>Player Fatal Injuries</English>
         </Key>
-        <Key ID="STR_ACE_Medical_Statemachine_FatalInjuryCondition_Description">
-            <English>Controls when a player can receive a fatal injury.</English>
-            <German>Definiert wann ein Spieler tödliche Verletzungen bekommen kann</German>
-            <Japanese>プレイヤーが致命傷から蘇生できるかどうか決定します</Japanese>
-            <Russian>Определяет, когда игрок может получить смертельное ранение</Russian>
-            <French>Définit quand un joueur peut recevoir une blessure fatale.</French>
+        <Key ID="STR_ACE_Medical_Statemachine_FatalInjuriesPlayer_Description">
+            <English>Controls when players can receive fatal injuries. A fatal injury is caused by receiving an injury to the head or heart.</English>
```

Comment:
This may need to be reworked particularly if we do the amassed wounds thing. I guess just 
> A fatal injury is caused by significant damage to the head or body

encompasses the cases of both organ damage (headshot / heartshot) or amassed wounding.